hoot-services/src/main/java/hoot/services/controllers/ingest/ImportCommand.java

```diff
@@ -88,18 +89,33 @@
             //options.add("schema.translation.script=" + translationPath);
         //}
 
-        if (!isNoneTranslation && (classification == SHP) || (classification == FGDB) || (classification == ZIP)) 
+        if (!isNoneTranslation && (classification == SHP) || (classification == FGDB) || (classification == ZIP))
         {
           options.add("schema.translation.script=" + translationPath);
         }
 
+        if (advUploadOpts != null && (classification == SHP)) {
+
+            List<String> getAdvOpts = Arrays.asList(advUploadOpts.split(","));
+
+            final boolean simplifyBuildings = getAdvOpts.stream().anyMatch("Ogr2osmSimplifyComplexBuildings"::equalsIgnoreCase);
+            final boolean mergeNearbyNodes  = getAdvOpts.stream().anyMatch("Ogr2osmMergeNearbyNodes"::equalsIgnoreCase);
+
+            if (simplifyBuildings) {
```

Comment:
This needs to be split up into two unique if blocks.
Even when both options are present, only the simplify options buildings will get added because (as the first) it will be true, pass through and add the options, then the else if will get ignored even though the option was passed

Reply:
I think we want advanced upload options to work like advance conflate options here
https://github.com/ngageoint/hootenanny/blob/master/hoot-services/src/main/java/hoot/services/controllers/conflation/ConflateCommand.java#L211-L224
No hardcoded options names, just loop through the submitted entries and if they exist in the known upload options then add them.

We also want to use these options for all input formats, not just SHP.  I think worse case the convert command will ignore those options not applicable.  I don't think we have the logic to present per format options in the UI.

Reply:
I re-read the original ticket, and see it does refer to OGR import formats, which would be more than just SHP.  I don't think we advertised the service endpoint as OGR options though.  Need to find out if there are OSM only options as well.